cms/static/js/views/paging.js

```diff
@@ -0,0 +1,143 @@
+;(function (define) {
```

Comment:
Yeah, I'd love to preserve history if possible, but I'm not sure exactly how to do that. I'd love to get your eyes on it if you have a moment.